### PR TITLE
Docs cruft and #3695

### LINKED ISSUE #3695: Server Example Out of Date
This server plot embedding example is out of date: http://bokeh.pydata.org/en/latest/docs/user_guide/embed.html#server-data

### PR DESCRIPTION
**Pull request by birdsarah**
Fixes #3695 

Continues work from #3688

### CODE REVIEW

**Comment by birdsarah** [comment]
hands off @bryevdv this isn't finished :D

**Comment by birdsarah** [comment]
@bryevdv this is ready - have at it :D

File: sphinx/source/docs/user_guide/embed.rst [hunk 1]
```diff
@@ -285,8 +285,7 @@ server data
 
 The simplest case is to use the Bokeh server to persist your plot and data.
 Additionally, the Bokeh server affords the opportunity of animated plots or
-updating plots with streaming data. The |autoload_server| function accepts
-a plot object and a Bokeh server ``Session`` object. It returns a ``<script>``
+updating plots with streaming data. The |autoload_server| returns a ``<script>``
```
Comment (bryevdv) [inline sphinx/source/docs/user_guide/embed.rst]: `The |autoload_server| function returns`

**Comment by birdsarah** [comment]
@bryevdv  - ok to merge when test passes?

**Pull request merged**